server/models/Wallet.js

```diff
@@ -415,7 +415,7 @@ class Wallet{
         if(
           trustRelationship.actor_entity_id === senderWallet.getId() &&
           trustRelationship.target_entity_id === receiveWallet.getId() &&
-          trustRelationship.request_type === TrustRelationship.ENTITY_TRUST_REQUEST_TYPE.send
+          trustRelationship.request_type === trustType
```

Comment:
@dadiorchen I noticed that this hasTrust method in wallet model was not actually referencing the trustType parameter sent in, so I changed it to be more flexible to reuse for checking other request types